src/scripts/pages/about/contact/contact-template.html

```diff
@@ -15,7 +15,7 @@
 
   <p>
     <strong>Phone:</strong> 713-348-5012<br />
-    <strong>Email:</strong> cnx@cnx.org
+    <strong>Email:</strong> support@openstax.org
```

Comment:
In our last QA round, @kajalp mentioned that this should be clickable email address. Can we make it a mailto item?